tests/MongoDB.Driver.Tests/MongoDB.Driver.Tests.csproj

```diff
@@ -57,14 +57,28 @@
     <ProjectReference Include="..\MongoDB.Driver.Core.TestHelpers\MongoDB.Driver.Core.TestHelpers.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Remove="Specifications\**\testsdata\**\*.json" />
+  </ItemGroup>
+  
+  <ItemGroup>
+    <EmbeddedResource Include="Specifications\**\testsdata\**\*.json" />
```

Comment:
`testsdata` folder can be removed. But in this case, we need to add `EmbeddedResource` attributes to json files which are out of `tests` folder manually

Reply:
From my perspective, I don't see big problems with working with the current solution. For this case, we need just to skip a top-level folder for comparing. Like this: 
![image](https://user-images.githubusercontent.com/44870738/63890695-2e6ef800-c9ec-11e9-88f8-494dd2f96bfe.png)

Reply:
I found this directory structure very confusing. It caused me to assume that all JSON files were JSON driven tests.

I propose the following structure:

    prose-tests
        files
            ... JSON input files to prose tests
        ClientEncryptionProseTests.cs
    tests
        ... JSON driven test files
    ClientSideEncryptionTestRunner.cs (reads JSON driver tests from the `tests` directory just like all other test runners)

This isolates all the *prose* tests in a single directory and makes it clear that *those* JSON files are not to be consumed by the customary JSON driven test runner.

Reply:
>I found this directory structure very confusing. It caused me to assume that all JSON files were JSON driven tests

Isn't the main reason for this assumption that we didn't use json files for any other target except of `json` files?

As far as I see, your current suggestion contradicts the idea to have the json files structure as close as possible to `spec` files, isn't it? I know that my implementation for the folder structure is also not fully the same as the spec. But I guess working/comparing/make synchronization with one more additional top folder is easier than with completely different folders structure?